pkg/controller.v1beta1/experiment/experiment_controller_test.go

```diff
@@ -89,7 +91,10 @@ func (statusM statusMatcher) String() string {
 
 func TestAdd(t *testing.T) {
 	g := gomega.NewGomegaWithT(t)
-	mgr, err := manager.New(cfg, manager.Options{Metrics: metricsserver.Options{BindAddress: "0"}})
+	mgr, err := manager.New(cfg, manager.Options{
+		Metrics:    metricsserver.Options{BindAddress: "0"},
+		Controller: config.Controller{SkipNameValidation: ptr.To[bool](true)},
```

Comment:
```suggestion
		Controller: config.Controller{SkipNameValidation: ptr.To(true)},
```

We should rely on the type assertion. Same comments on other places as well.

Reply:
I'll udpate them soon:)